plugins/chat/assets/javascripts/discourse/services/chat-channels-manager.js

```diff
@@ -130,11 +129,7 @@ export default class ChatChannelsManager extends Service {
         channel.isCategoryChannel && channel.currentUserMembership.following
     );
 
-    if (this.site.mobileView) {
-      return this.#sortChannelsByActivity(channels);
-    } else {
-      return channels.sort((a, b) => a?.slug?.localeCompare?.(b?.slug));
-    }
+    return this.#sortChannelsByActivity(channels);
```

Comment:
Where the main change happens for channel sorting, using same activity order as mobile.